docs/build-environment.yml

```diff
@@ -14,7 +14,7 @@ dependencies:
   - yarn
   - jupyterlab >=3.5.3,<3.6
   - jupyterlite-core >=0.1.0,<0.2.0
-  - empack >=3,<4
+  - empack >=3.1.0
```

Comment:
No need to keep the upper bound?

Reply:
That's just the dev environment so that's not too bad. Probably good to catch empack compatibility early